cfgov/unprocessed/css/organisms/video-player.less

```diff
@@ -153,6 +153,38 @@
         bottom: unit( @grid_gutter-width / 2 / @play-btn-icon-size, em );
     }
 
+    &__center,
+    &__bottom-right {
+        &:focus-visible {
```

Comment:
Both Safari and IE don't support `focus-visible`. Would you see any issue with putting…

```
&:focus-visible,
&:focus {
```

…here to catch IE?

(I can't figure out Safari).

Reply:
I don't see there being any issue with it, let me test though